ui.frontend/src/main/styles/commerce/_productlist.scss

```diff
@@ -174,7 +174,7 @@
     background: none;
     border: 0;
     cursor: pointer;
-    font-family: var(--venia-font);
+    font-family: var($venia-font);
```

Comment:
This needs to be `font-family: $venia-font;`. Don't mix CSS and SCSS variables.

Reply:
Good catch, I fixed it (was correct in the initial PR from Markus).